src/components/recaptcha/editForm/ReCaptcha.edit.display.js

```diff
@@ -0,0 +1,103 @@
+export default [
+  {
+    key: 'eventType',
+    label: 'Type of event',
+    tooltip: 'Specify type of event that this reCAPTCHA would react to',
+    type: 'radio',
+    values: [
+      {
+        label: 'Form Load',
+        value: 'formLoad'
+      },
+      {
+        label: 'Button Click',
```

Comment:
If this is attached to the Submit button, this component will need to block the submission until it finishes the recaptcha check and then allow the submission to be completed.

Reply:
@travist fixed